blocks/library/table/table-block.js

```diff
@@ -3,9 +3,28 @@ import BlockControls from '../../block-controls';
 import { Toolbar, DropdownMenu } from 'components';
 import { __ } from 'i18n';
 
+function isTableSelected( editor ) {
+	return editor.dom.getParent(
+		editor.selection.getStart( true ),
+		'table'
```

Comment:
Do we know in this case that the table ancestor here is from _this_ table block, and not another table on the page?